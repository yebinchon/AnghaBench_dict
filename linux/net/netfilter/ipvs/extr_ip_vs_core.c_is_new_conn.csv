SCTP_CID_INIT,VAR_0
skb_header_pointer,FUNC_0
is_new_conn,FUNC_1
skb,VAR_1
iph,VAR_2
_tcph,VAR_3
th,VAR_4
sch,VAR_5
schunk,VAR_6
