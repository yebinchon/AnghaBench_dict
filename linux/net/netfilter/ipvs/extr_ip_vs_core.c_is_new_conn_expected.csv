IP_VS_CONN_F_NOOUTPUT,VAR_0
IP_VS_SCTP_S_CLOSED,VAR_1
IP_VS_TCP_S_CLOSE,VAR_2
IP_VS_TCP_S_FIN_WAIT,VAR_3
IP_VS_TCP_S_TIME_WAIT,VAR_4
is_new_conn_expected,FUNC_0
cp,VAR_5
conn_reuse_mode,VAR_6
